custom_model_runner/datarobot_drum/drum/server.py

```diff
@@ -22,13 +22,16 @@ def get_flask_app(api_blueprint):
     return app
 
 
-def base_api_blueprint(termination_hook=None):
+def base_api_blueprint(termination_hook=None, predictor=None):
     model_api = Blueprint("model_api", __name__)
 
     @model_api.route("/", methods=["GET"])
     @model_api.route("/ping/", methods=["GET"], strict_slashes=False)
     def ping():
         """This route is used to ensure that server has started"""
+        if hasattr(predictor, "liveness_probe"):
+            return predictor.liveness_probe()
+
```

Comment:
@zohar-mizrahi I added the `liveness_probe` to the `/ping` route and the `readiness_probe` to the `/health` route.